source/slang/slang-session.cpp

```diff
@@ -1432,6 +1471,34 @@ RefPtr<Module> Linkage::findOrImportModule(
         return glslModule;
     }
 
+    // Special case for neural module - search for it relative to libslang.so location
+    if (moduleName->text == "neural")
```

Comment:
I am thinking maybe we should generalize this logic (and the glsl case too) to look for builtin module libraries when the standard means of module discovery fails. This will make sure we don't break existing applications if there is a user-defined `neural` module.